L1Trigger/L1TTrackMatch/python/L1TrackerEtMissProducer_cfi.py

```diff
@@ -1,11 +1,10 @@
 import FWCore.ParameterSet.Config as cms
-from L1Trigger.VertexFinder.VertexProducer_cff import VertexProducer
-from L1Trigger.L1TTrackMatch.L1TrackSelectionProducer_cfi import L1TrackSelectionProducer, L1TrackSelectionProducerExtended
+from L1Trigger.L1TTrackMatch.L1TrackSelectionProducer_cfi import l1tTrackSelectionProducer, l1tTrackSelectionProducerExtended
```

Comment:
Here, there is still a dependency between `cfi`s. Below, `L1TrackSelectionProducer.outputCollectionName.value() + "Emulation"` could/should be replaced with the explicit name of the collection.

Reply:
Done